src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
```

Comment:
The universal selector '*' is used here which is not recommended for performance reasons. It is better to apply styles to specific elements or classes. You can find this in the checklist: 'Don't use * selector (it impacts performance)'.

Reply:
Which selector would you recommend using then? If instead of "*" you use "html" or "body" selectors, the tests will not pass